src/schema/v2/filterArtworksConnection.ts

```diff
@@ -392,7 +392,14 @@ const filterArtworksConnectionTypeFactory = (
       keyword_match_exact: keywordMatchExact,
       ..._options,
     }
-    const { include_artworks_by_followed_artists, aggregations } = options
+    const {
+      first,
+      last,
+      after,
+      before,
+      include_artworks_by_followed_artists,
```

Comment:
Are you sure this is still valid? As far as I know all the snake case references were removed.

Reply:
Yea, if you look closer this is destructuring out of `options`, which already does the camel case (incoming args) -> snake case (for backend API calls) translation.

Reply:
oooooohhh, right. Man, that always gets me.